tools/shared.py

```diff
@@ -2672,7 +2672,8 @@ def minify_wasm_imports_and_exports(js_file, wasm_file, minify_whitespace, minif
     logger.debug('minifying wasm imports and exports')
```

Comment:
It looks like we already skip this for SIDE_MODULE.  Presumably for this change to work we should also skip for MAIN_MODULE.   In other words skip if RELOCATABLE (which is set for both side and main module).

Reply:
Ok, I have turned off inter wasm and js minification for dynamic linking altogether.